Core/Core/View/Base/ResponsiveView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ResponsiveView.swift
+//  Core
+//
+//  Created by  Stepanok Ivan on 26.03.2024.
+//
+
+import SwiftUI
+
+public struct ResponsiveView: View {
```

Comment:
Is it possible to give it a meaningful name as every view is by default responsive view.

Reply:
Renamed to DynamicOffsetView. Good?